### PR TITLE
Backfill tests

### PR DESCRIPTION
**Pull request by daytonn**
@kand @collin Just adding some tests for makeRequest.

### CODE REVIEW

**Comment by collin** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bulbs-elements",
-  "version": "0.2.5",
+  "version": "0.3.0-rc1",
```
Comment (daytonn) [inline package.json]: Not sure why this guy snuck in but I'm not sure it matters

File: lib/bulbs-elements/util/make-request.js [hunk 2]
```diff
@@ -2,8 +2,8 @@ import cloneObject from 'clone';
 import invariant from 'invariant';
 
 export default function makeRequest (url, options = {}) {
-  invariant(options.success, 'makeRequest MUST have an success callback');
-  invariant(options.failure, 'makeRequest MUST have an failure callback');
+  invariant(options.success, 'makeRequest MUST have a success callback');
+  invariant(options.failure, 'makeRequest MUST have a failure callback');
```
Comment (spra85) [inline lib/bulbs-elements/util/make-request.js]: :+1: to proper grammar

**Comment by spra85** [comment]
:+1:

File: lib/bulbs-elements/util/make-request.js (hunk 2 above)
Comment (collin) [inline lib/bulbs-elements/util/make-request.js]: # Grammanarchy

**Pull request merged**